atomic_reactor/plugins/post_push_to_pulp.py

```diff
@@ -109,47 +152,32 @@ def push_tar(self, filename, image_names=None, repo_prefix="redhat-"):
 
         pulp_repos = self.pulp_handler.create_dockpulp_and_repos(image_names, repo_prefix)
         _, file_extension = os.path.splitext(filename)
+        upload_filename = None
```

Comment:
Nitpick: `upload_filename` doesn't seem right. I think something like `compressed_filename` is what you want.

Reply:
ok